src/wincurse.cpp

```diff
@@ -591,9 +591,7 @@ uint64_t GetPerfCount(){
     return Count;
 }
 
-//Not terribly sure how this function is suppose to work,
-//but jday helped to figure most of it out
-int curses_getch(WINDOW* win)
+input_event input_manager::get_input_event( WINDOW *win )
```

Comment:
Fogot to #include "input.h" in tis file, I'll fix up in the merge unless something else breaks.